### ISSUE TITLE
Kp scalar on distribution tab

### ISSUE THREAD

**Issue by fredsevmartins** [Open-Systems-Pharmacology/PK-Sim#1107]
**Kp scalar on distribution tab**

The Kp scalar factor on distribution tab is missed when project is saved and the PKsim shut down .  By default kp scalar is 1 to all methods, but if you replace it by a user value  , for example, 0.1 click and scale then save the project and close the PKsim when you open again the number in the box will be 1 and not 0.1 anymore.

**Comment by fredsevmartins** [comment]
> @fredsevmartins The scalar is just a number that allows you to scale the visible parameters by multiplying them with a given factor. It never indicates how the factor were scaled. For example, you can set the factor to 0.1 and scale the parameters twice. Resulting in an effective scaling by 0.01 but the value displayed will always be 0.1.
> 
> In my opinion, nothing to change here. Does it make sense to you?

**Comment by fredsevmartins** [comment]
> > @fredsevmartins The scalar is just a number that allows you to scale the visible parameters by multiplying them with a given factor. It never indicates how the factor were scaled. For example, you can set the factor to 0.1 and scale the parameters twice. Resulting in an effective scaling by 0.01 but the value displayed will always be 0.1.
> > In my opinion, nothing to change here. Does it make sense to you?

@msevestre  Thank you very much for your prompt feedback. The new value never is loaded when I open my model in second time. PLease see the figure attached the value should be 0.1.
![image](https://user-images.githubusercontent.com/45317633/53120188-929e9400-3551-11e9-92b5-80fbd633d8b5.png)

**Comment by msevestre** [comment]
Yes the parameter values are set. Just the factor isn't as it could very easily show something wrong

1/ As mentioned earlier, if you set the SCALE factor to 0.1 and press `SCALE` twice, the value of the Kp parameters would not match the scaled factor use (x0.0.1 vs  x0.1)

2/You could also scale the values and then update one KP parameter by hand, which would result in the scale factor not matching the values of your parameters

So in a nutshell, see this scale factor as a one and forget parameter. It does not influence the model whatsoever and is simply a convenience method to allow you to scale all visible parameters by the same value at once

**Comment by msevestre** [comment]
@fredsevmartins Closing the issue for now. Please reopen if you need more clarification
